sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -443,6 +446,31 @@ case class AlterTableDropPartitionCommand(
 case class AlterTableRecoverPartitionsCommand(
     tableName: TableIdentifier,
     cmd: String = "ALTER TABLE RECOVER PARTITIONS") extends RunnableCommand {
+
+  // These are two fast stats in Hive Metastore
```

Comment:
Let's add a little more context here so that this comment is self explanatory -- perhaps something along the lines of "These are list of statistics that can be collected quickly without requiring a scan of the data"